test/e2e/declarative_test_fwk/runner.py

```diff
@@ -193,17 +194,38 @@ def assert_expectations(step: model.Step) -> None:
     """
 
     logging.info(f"assert:  {step}")
-    if not step.expectations:
+    if not step.expectations_k8s:
         return
 
     resource_helper = helper.get_resource_helper(step.resource_kind)
     reference = resource_helper.custom_resource_reference(step.input_data, step.replacements)
     try:
-        resource_helper.assert_expectations(step.verb, step.input_data, step.expectations, reference)
+        resource_helper.assert_expectations_k8s(step.verb, step.input_data, step.expectations_k8s, reference)
     except AssertionError as ae:
         logging.error(f"AssertionError at {step}")
         raise ae
 
+def assert_api_response(step: model.Step) -> None:
+    """Asserts api_response as specified in the Step.
+
+    Args:
+        step: test step
+
+    Returns:
+        None
+    """
+
+    logging.info(f"assert:  {step}")
```

Comment:
Is this logging meant to be here?

Reply:
Perhaps this should become `logging.debug`?

Reply:
I've changed this one to `logging.debug`. However, `runner.py file` has both `logging.info` and `logging.debug` inside. Should I change all `logging.info` to `logging.debug`? I feel like either one is fine, but I want to keep them consistent.